### PR TITLE
moyo header

### PR DESCRIPTION
**Pull request by Shparhala**
- [DEMO LINK](https://Shparhala.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Shparhala.github.io/layout_moyo-header/report/html_report/)

+ Header height is set in 1 place (for the links)
+ Content is vertically centered (for any header height)
+ CSS is used to show all letters in Uppercase (don't type them in HTML)
+ Logo is an image wrapped with a link
 + CSS Variable is used for a blue color
 + Pseudo-element is used for a blue line below the active link
 + Code follows all the [Code Style Rules ❗️](https://github.com/Shparhala/layout_moyo-header/blob/master/checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,39 @@
...
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item "><a href="#apple" class="nav__link is-active">Apple</a></li>
+          <li class="nav__item"><a href="#samsung" class="nav__link">Samsung</a></li>
+          <li class="nav__item"><a href="#smartphones" class="nav__link">Smartphones</a></li>
+          <li class="nav__item">
+            <a href="#laptops&computers"
+               class="nav__link #4"
+               data-qa="hover"
+             >Laptops & computers
```
Comment (etojeDenys) [inline src/index.html:33]: Pasted Graphic

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,100 @@
...
+    max-height: 60px;
+    bottom: 0;
+    }
+
+  .is-active::after {
+  position: absolute;
+  bottom: 0;
+  content: "";
+  display: block;
+  background-color: #00ACDC;
```
Comment (etojeDenys) [inline src/style.css:93]: move this color to some var and reuse everywhere

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (Shparhala) [reply src/index.html]: corrected

File: src/style.css (hunk 2 above)
Comment (Shparhala) [reply src/style.css]: corrected

**Comment by Shparhala** [comment]
Неможу зрозуміти, що з перевірками автоматичними... Я лише виправив помилки, які вказані.

**Pull request closed without merging**